backend/sass/common.blocks/_wallet.scss

```diff
@@ -19,12 +19,11 @@
 
 .wallet__table-heading {
   @extend .table__heading;
-  padding: $medium-padding;
-  padding-bottom: $bigger-padding;
+  padding: 15px 30px 30px 15px;
 }
 
 .wallet__table-cell {
-  padding: $medium-padding;
+  padding: 15px 30px 15px 15px;
```

Comment:
Right padding is greater than left padding?

Reply:
Yeah. The rejected ticket was explicitly mentioning the fact that there wasn't enough padding between the keys. This way being lazy to get that effect, but you are right that it is confusing and I've done it a better way in 83dce35